src/components/OrderList/OrderList.js

```diff
@@ -66,8 +58,15 @@ function OrderList({ filter }) {
             <td>
               {order.buyer.firstName} {order.buyer.lastName}
             </td>
-            <td>Rose Necklace</td>
-            <td>{order.quantity}</td>
+            <td>
+              {order.items.map((item) => (
+                <>
+                  <label className="my-0" style={{ display: "block" }}>
+                    {item.product.name} (x{item.quantity})
+                  </label>
```

Comment:
```suggestion
                  <div className="my-0">
                    {item.product.name} (x{item.quantity})
                  </div>
```
You can use a block level element.

Reply:
oh 😮. I didn't thought about that. 👍